tests/playwright/factories/handlers/personalDetailsHandlers.ts

```diff
@@ -0,0 +1,82 @@
+/**
+ * Personal Details endpoint handlers
+ */
+
+import { http, HttpResponse } from 'msw';
+import type { MockCase } from './types.js';
+import { transformToApiFormat } from './utils.js';
+import {
+  validateStringField,
+  validateNullableBooleanField,
+  validateChoiceField
+} from './validationHelpers.js';
+
+export function createPersonalDetailsHandlers(
+  API_BASE_URL: string,
+  API_PREFIX: string,
+  cases: MockCase[]
+) {
+  return [
+    // PATCH /case/:caseReference/personal_details/
+    http.patch(`${API_BASE_URL}${API_PREFIX}/case/:caseReference/personal_details/`, async ({ params, request }) => {
+      const { caseReference } = params;
+      const updateData = await request.json() as Record<string, any>;
+      
+      const caseItem = cases.find(c => c.caseReference === caseReference);
+      
+      if (!caseItem) {
+        return HttpResponse.json({ error: 'Case not found' }, { status: 404 });
+      }
+
+      // Validate request data structure
+      const validationErrors: Record<string, string[]> = {};
+
+      if (typeof updateData !== 'object' || updateData === null || Array.isArray(updateData)) {
+        return HttpResponse.json({ detail: 'Invalid request body format' }, { status: 400 });
+      }
+
+      // Validate string fields with max length
+      validateStringField(updateData, 'full_name', 400, validationErrors);
```

Comment:
I don't think any of our validations, care about the character length. Are we adding in extra, which doesn't represent our service?